common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -72,7 +72,10 @@ innovation_giants = {
 	icon = @giants
 
 	potential = {
-		can_unlock_innovation_giants_trigger = yes
```

Comment:
Why it's gone?
Also, `common/scripted_triggers/wc_maa_triggers.txt` got reverted too? I checked it happened in a merge.

Reply:
If I click 'Delete File' in the files changes tab on this PR, will it just revert my changes? If so can I just do that?